src/monogame/include/stl_common.h

```diff
@@ -0,0 +1,21 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+#pragma once
+
+
+#include <assert.h>
+#include <vector>
+#include <string>
+#include <queue>
+#include <map>
+
+
+template <class T>
+void remove_by_value(std::vector<T>& vector, const T& element)
```

Comment:
For now i'm depending on STL for vectors, string, queue, maps, etc.

I have seen differences in functionality and performance across platforms over the years.

I wonder if eventually we want our own small common set of native helper collections to use across our native backends?

Reply:
I think we should only implement our own if we come across an issue with performance.